src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -27,6 +29,7 @@ const initialState: Preferences = {
   },
   selectedLocale: simpleLocale(),
   balanceThreshold: '0',
+  selectedCurrency: 'USD',
```

Comment:
required for language selection, not yet used in view layer, needed to bring over selectors from monster branch without rewriting